src/NuGet.Core/NuGet.Commands/NuGet.Commands.csproj

```diff
@@ -13,6 +13,7 @@
     <Shipping>true</Shipping>
     <XPLATProject>true</XPLATProject>
     <UsePublicApiAnalyzer>perTfm</UsePublicApiAnalyzer>
+    <WarningsNotAsErrors>$(WarningsNotAsErrors);IDE0055</WarningsNotAsErrors>
```

Comment:
The reason why these warnings are suppressed in this project is the fact that we're using submodules and not the packages, https://github.com/NuGet/Home/issues/2345, so we don't have *control* over all of the code.